src/components/Card/Card.js

```diff
@@ -38,7 +39,14 @@ const Card = (props) => {
   // const styles = buildStylesheet('card', props.styles);
   const renderImageComponent = () => {
     if (image)
-      return <Image testID="contentLinkImageUrlImage" style={styles.image} source={{uri: image}} />;
+      return (
+        <Image
+          testID="contentLinkImageUrlImage"
+          style={styles.image}
+          source={{uri: image}}
+          resizeMode="cover"
+        />
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `resizeMode` prop has been added to the `Image` component. This is a good change as it allows for more control over how the image is displayed. However, it's important to note that using "cover" may cause some parts of the image to not be visible if the aspect ratio of the image differs from the aspect ratio of the Image component. If this is not the desired behavior, consider changing the resizeMode to "contain", which will ensure that the entire image is visible, albeit with potential letterboxing.



<!-- This is an auto-generated comment by OSS CodeRabbit -->